### PR TITLE
exodrone adventures are no longer database based

### PR DESCRIPTION
**Pull request by mc-oofert**
## About The Pull Request

you now put the jsons into the folder at strings/exoadventures
no adventures have been lost i think
also deletes the example_adventures folder since theyre just duplicates of the adventures in the other folder now

## Why It's Good For The Game

having it database based meant you had to ping someone with database access to add it which isnt exactly optimal for contributions

## Changelog
:cl:
code: exploration drone adventures are now file-based and not database-based
/:cl:

### CODE REVIEW

File: SQL/database_changelog.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ Any time you make a change to the schema files, remember to increment the databa
 
 Make sure to also update `DB_MAJOR_VERSION` and `DB_MINOR_VERSION`, which can be found in `code/__DEFINES/subsystem.dm`.
 
-The latest database version is 5.24; The query to update the schema revision table is:
+The latest database version is 5.25; The query to update the schema revision table is:
 
 ```sql
 INSERT INTO `schema_revision` (`major`, `minor`) VALUES (5, 24);
```
Comment (san7890) [inline SQL/database_changelog.md:8]:
```suggestion
INSERT INTO `schema_revision` (`major`, `minor`) VALUES (5, 25);
```

there's another one a few lines down too

**Review by san7890 — CHANGES_REQUESTED** [review]
would you mind prettyprinting some of the JSON here. i despise the one that's two lines that are five trillion characters long

**Comment by mc-oofert** [comment]
> would you mind prettyprinting some of the JSON here. i despise the one that's two lines that are five trillion characters long

youre not supposed to edit the jsons

https://anturk.github.io/adventure-maker/

**Comment by san7890** [comment]
how the flarp does pretty printing affect anything

File: SQL/database_changelog.md (hunk 1 above)
Comment (san7890) [reply SQL/database_changelog.md]: you missed the other one a few lines down

**Comment by mc-oofert** [comment]
> how the flarp does pretty printing affect anything

its sisyphean to even do anything outside of changing a number on an adventure from the json

**Comment by san7890** [comment]
> > how the flarp does pretty printing affect anything
> 
> its sisyphean to even do anything outside of changing a number on an adventure from the json

yes that's why it needed it

**Pull request merged**